config/dropt/grass.json

```diff
@@ -1,30 +0,0 @@
-{
-  "rules": [
-    {
-      "match": {
-        "blocks": {
-          "blocks": [
-            "minecraft:tallgrass:*"
-          ]
-        },
-        "harvester": {
-          "type": "PLAYER",
-          "heldItemMainHand": {
-            "items": [
-				"notreepunching:knife/flint"
-			]
-          }
-        }
-      },
-      "drops": [
-		{
-			"item": {
-				"items": [
-					"notreepunching:grass_fiber"
-				]
-			}
```

Comment:
we are using these scripts ig?

Reply:
Well, i didn't find a mod called dropt so i considered them redundant/deprecated

Reply:
its cf code is 284973, still in the manifest